src/database/Database.cpp

```diff
@@ -288,7 +288,7 @@ Database::initialize()
     HerderPersistence::dropAll(*this);
     mApp.getLedgerStateRoot().dropData();
     BanManager::dropAll(*this);
-    putSchemaVersion(1);
+    putSchemaVersion(6);
```

Comment:
you should keep this change in the previous commit so that you don't even need to special case ` if (vers <= 6)` (and error in that case, which is also missing from this changeset)